packages/repository-tests/src/crud/relations/fixtures/repositories/address.repository.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Getter} from '@loopback/context';
+import {
+  BelongsToAccessor,
+  juggler,
+  createBelongsToAccessor,
+  BelongsToDefinition,
+} from '@loopback/repository';
+import {Address, AddressRelations, Customer} from '../models';
+import {CrudRepositoryCtor} from '../../../..';
+
+export function createAddressRepo(repoClass: CrudRepositoryCtor) {
+  return class AddressRepository extends repoClass<
+    Address,
+    typeof Address.prototype.id,
+    AddressRelations
+  > {
+    public readonly customer: BelongsToAccessor<
+      Customer,
+      typeof Address.prototype.id
+    >;
+
+    constructor(
+      db: juggler.DataSource,
+      customerRepositoryGetter: Getter<typeof repoClass.prototype>,
+    ) {
+      super(Address, db);
+      // cant use the method createHasManyRepositoryFactoryFor from DefaultCrud
+      // cause it's protected.
```

Comment:
Not only the method `createHasManyRepositoryFactoryFor` is protected in the default repository class, but you are also inheriting from `repoClass` which may not provide `createHasManyRepositoryFactoryFor` at all. 

I feel the current comment can be a bit misleading, could you please improve it?